api/src/main/java/org/openmrs/module/attachments/AttachmentsContext.java

```diff
@@ -196,6 +210,43 @@ protected Boolean getBooleanByGlobalProperty(String globalPropertyName) {
 		return BooleanUtils.toBoolean(globalProperty);
 	}
 	
+	protected Concept getConceptByGlobalProperty(String globalPropertyName) {
+		String globalProperty = administrationService.getGlobalProperty(globalPropertyName);
+		Concept concept = conceptService.getConceptByUuid(globalProperty);
+		if (concept == null) {
+			throw new IllegalStateException("Configuration required: " + globalPropertyName);
+		}
+		return concept;
+	}
+	
+	protected EncounterType getEncounterTypeByGlobalProperty(String globalPropertyName, boolean required) {
+		String globalProperty = administrationService.getGlobalProperty(globalPropertyName);
+		EncounterType encounterType = encounterService.getEncounterTypeByUuid(globalProperty);
+		if (required && encounterType == null) {
+			throw new IllegalStateException("Configuration required: " + globalPropertyName);
+		}
+		return encounterType;
+	}
+	
+	protected Integer getIntegerByGlobalProperty(String globalPropertyName) {
```

Comment:
It seems like this could return an `int` and use `Integer#parseInt` right?

Reply:
Ok...